test/com/amazon/ion/impl/bin/IonEncoder_1_1Test.java

```diff
@@ -144,8 +158,8 @@ public void testWriteIntegerValue(long value, String expectedBytes) {
             "          -9223372036854775809, F5 13 FF FF FF FF FF FF FF 7F FF",
             "-99999999999999999999999999999, F5 1B 01 00 00 60 35 E8 8D 92 51 F0 E1 BC FE",
     })
-    public void testWriteIntegerValueForBigInteger(String value, String expectedBytes) {
-        assertWritingValue(expectedBytes, new BigInteger(value), IonEncoder_1_1::writeIntValue);
+    public void testWriteIntegerValueForBigInteger(BigInteger value, String expectedBytes) {
+        assertWritingValue(expectedBytes, value, IonEncoder_1_1::writeIntValue);
```

Comment:
This is also an incidental change since I realized that JUnit already knows how to convert to `BigInteger`.